yellow-paper/sidebars.js

```diff
@@ -185,19 +185,22 @@ const sidebars = {
       ],
     },
     {
-      label: "Public VM",
+      label: "Aztec (Public) VM",
       type: "category",
-      link: { type: "doc", id: "public-vm/avm" },
+      link: { type: "doc", id: "public-vm/intro" },
       items: [
-        "public-vm/avm",
+        "public-vm/intro",
         "public-vm/state",
-        "public-vm/type-structs",
         "public-vm/memory-model",
+        "public-vm/context",
+        "public-vm/execution",
+        "public-vm/nested-calls",
         "public-vm/instruction-set",
         "public-vm/avm-circuit",
         "public-vm/control-flow",
         "public-vm/alu",
         "public-vm/bytecode-validation-circuit",
```

Comment:
Should the circuit related sections be under AVM Circuit? (or their own section under Public VM?)